packages/frontend/addresses/1337.ts

```diff
@@ -1,3 +1,3 @@
 export const ContractAddresses_1337 = {
-  "YieldGate": "0x5fbdb2315678afecb367f032d93f642f64180aa3"
-}
\ No newline at end of file
+  YieldGate: '0x5fbdb2315678afecb367f032d93f642f64180aa3',
```

Comment:
Would be good to also add a commit fixing the script `packages/hardhat/scripts/deploy.ts` (line 49) that creates these files, because they will always be formatted the wrong way otherwise. But happy to do it in a separate PR.